Makefile

```diff
@@ -19,6 +19,11 @@ COMMIT_SHA=$(shell git rev-parse --short HEAD)
 # Go and compilation related variables
 BUILD_DIR ?= out
 
+# Docs build related variables
+DOCS_BUILD_DIR ?= $(CURDIR)/docs/build
```

Comment:
this should be without `$(CURDIR)` ?

Reply:
Will fix this and add `:Z` option for bind mount, and merge it!